### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Avin008**
### Landing Page Includes: 

landing

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+          <div class="jumbotron__CTA">
+            <a
+              href="#documentation"
+              class="jumbotron__btn jumbotron__btn--primary"
+              >Get Started</a
+            >
+            <a
+              href="/Docs/docs.html"
+              class="jumbotron__btn jumbotron__btn--secondary"
```
Comment (Manju-sree) [inline index.html:66]: same here

**Review by Manju-sree — CHANGES_REQUESTED** [review]
better to use one type of class declarations in whole file. rest code looks fine..

File: index.html (hunk 1 above)
Comment (Avin008) [reply index.html:66]: Here I'm using jumbotron__btn--secondary as a modifier class. Can you plz approve the changes

Comment (Manju-sree) [reply index.html:66]: okay, it's looks fine now, you can go ahead with merge

**Review by Manju-sree — APPROVED** [review]
LGTM

**Pull request merged**